### PR TITLE
ISSUE-870 FirebaseSubscription/get should return expired subscriptions

### PR DESCRIPTION
**Pull request by quantranhong1999**

### CODE REVIEW

File: tmail-backend/jmap/extensions/src/main/java/com/linagora/tmail/james/jmap/firebase/FirebaseSubscriptionHelper.java [hunk 1]
```diff
@@ -27,7 +27,7 @@ public static FirebaseSubscriptionExpiredTime evaluateExpiresTime(Optional<Zoned
             .orElse(maxExpiresTime));
     }
 
-    public static boolean isNotOutdatedSubscription(FirebaseSubscription subscription, Clock clock) {
-        return subscription.expires().isAfter(ZonedDateTime.now(clock));
+    public static boolean isNotOutdatedSubscription(FirebaseSubscription subscription) {
+        return subscription.expires().isAfter(ZonedDateTime.now());
```
Comment (chibenwa) [inline tmail-backend/jmap/extensions/src/main/java/com/linagora/tmail/james/jmap/firebase/FirebaseSubscriptionHelper.java:31]: Idem keep the clock!

**Review by Arsnael — COMMENTED** [review]
Read it as well

**Comment by Arsnael** [comment]
https://james-jenkins.lin-saas.com/job/Tmail%20build/job/PR-872/1/testReport/junit/com.linagora.tmail.james/MemoryFirebasePushContractTest/shouldNotPushStateChangeWhenExpiredSubscription_GuiceJamesServer__UpdatableTickingClock_/

Likely an issue for removing the clock :)

**Review by Arsnael — APPROVED** [review]

**Review by chibenwa — APPROVED** [review]

**Review by vttranlina — APPROVED** [review]

**Comment by Arsnael** [comment]
@quantranhong1999 can you squash that fixup? :)

**Pull request merged**